docs/maintaining.md

```diff
@@ -0,0 +1,166 @@
+Once your Neurobagel node is running and configured correctly,
+there are some recurring tasks you may have to do to keep it operating correctly.
+
+## Updating the Neurobagel services
+
+### Updating the Neurobagel Docker images
+
+We are continuously improving Neurobagel tools and services, 
+so you may want to update your Neurobagel node to the latest version to benefit from new features and bug fixes.
+We always publish our tools as [Docker images on Dockerhub](https://hub.docker.com/repositories/neurobagel).
+
+Each Docker image has a version tag, and also two rolling tags: 
+
+- `latest` (the latest stable release). This is the default tag used in the Neurobagel `docker-compose.yml` file.
+- `nightly` (the latest build from the main branch). This tag is only used for compatibility testing and should not be used in production.
+
+You can pull the most recent docker images for Neurobagel tools by running:
+
+```bash
+docker compose --profile full_stack pull
+```
+
+!!! warning "`docker compose` will only pull the images that are used by the current deployment profile."
+
+    If you don't specify a deployment profile, the default profile (`local_node`) will be used,
+    which only pulls the images for the [API](api.md), and graph store.
+
+    See the [deployment profiles](infrastructure.md#deployment-profiles) 
+    section for more information on the available profiles.
+
+### Restarting services after an update
+
+Whether you have updated the Docker images, the [configuration](config.md), or the [data](#updating-the-data-in-your-graph) 
+of your Neurobagel node, you will need to restart the services to apply the changes.
+
+To shut down a running Neurobagel node, 
+navigate to the path on your file system where 
+you have stored the `docker-compose.yml` file from the [initial setup](getting_started.md) and run:
+
+```bash
+docker compose --profile full_stack down
+```
+
+Then, to start the services again:
+
+```bash
+docker compose --profile full_stack up -d
+```
+
+!!! note "Specify the deployment profile"
+    
+    Whenever you use `docker compose` commands, you also have to specify
+    the deployment profile you want to use with the `-p` or `--profile` flag.
+    If you forget this step, `docker compose` will use the default profile (`local_node`),
+    and will only shut down, update, or start the services for that profile.
+
+## Updating the data in your graph
```

Comment:
Because this section covers actually **re-annotating and re-harmonizing** data using the annotation tool and CLI, which would be targeted to a RA with dataset expertise rather than a sysadmin, I think it makes more sense to be just renamed "Updating a harmonized dataset" and moved to the "Annotating/harmonizing a dataset" section (also because at this point, we have not mentioned those tools at all).

The part I would leave on this page, under "Updating the data in your graph", is the section currently under the "Maintaining a node" page describing using `add_data_to_graph` to reupload datasets. Again, b/c this is something to be handled by the sysadmin.

Another reason I think this makes sense is because re-annotating and re-running the CLI on data may likely happen (and should be easy to do) asynchronously from when that data is actually pushed to the graph.